libimgui-docking/LICENSE.txt

```diff
@@ -0,0 +1 @@
+../upstream/LICENSE.txt
```

Comment:
Minor:
Currently this symlink references a file from `upstream`, however I think it should reference `upstream-docking`

Reply:
Uh, nice catch!